lib/spack/spack/test/cmd/versions.py

```diff
@@ -37,12 +38,31 @@ def test_remote_versions_only():
     versions('--remote', 'zlib')
 
 
-@pytest.mark.maybeslow
 @pytest.mark.usefixtures('mock_packages')
-def test_new_versions_only():
+def test_new_versions_only(monkeypatch):
     """Test a package for which new versions should be available."""
-
-    versions('--new', 'brillig')
+    def mock_fetch_remote_versions(*args, **kwargs):
+        mock_remote_versions = {
+            # new version, we expect this to be in output:
+            Version('99.99.99'): {},
+            # some packages use '3.2' equivalently to '3.2.0'
+            # thus '3.2.1' is considered to be a new version
+            # and expected in the output also
+            Version('3.2.1'): {},  # new version, we expect this to be in output
+            Version('3.2'): {},
+            Version('1.0.0'): {},
+        }
+        return mock_remote_versions
+    mock_versions = {
+        # already checksummed versions:
+        Version('3.2'): {},
+        Version('1.0.0'): {},
+    }
+    from spack.pkg.builtin.mock.brillig import Brillig
```

Comment:
Can we move this statement just under the docstring? Otherwise LGTM.

Reply:
done!